api/v1alpha1/nicclusterpolicy_types.go

```diff
@@ -106,6 +106,11 @@ type OFEDDriverSpec struct {
 	// +kubebuilder:default:=300
 	// +kubebuilder:validation:Minimum:=0
 	TerminationGracePeriodSeconds int64 `json:"terminationGracePeriodSeconds,omitempty"`
+	// Path to a persistent writable folder where OFED container can store its build artifacts.
+	// If the artifacts for the correct version and arch exist in this folder when the OFED container is started,
+	// they will be reused without driver being recompiled.
+	// +optional
+	DriverInventoryPath string `json:"driverInventoryPath,omitempty"`
```

Comment:
nit: OFED env variable contains "drivers". Do we want to rename the CR option to DriversInventoryPath WDYT?